lib/kitchen/directions/bake_index/v1.rb

```diff
@@ -168,6 +168,10 @@ def add_term_to_index(term_element, page_title, term_italicized)
         term_reference = term_element['cmlnle:reference'] || term_element['reference']
         group_by = term_reference[0]
         content = term_reference
+      elsif term_element['cxlxt:index'] == 'name' || term_element['index'] == 'name'
```

Comment:
I have extended the condition for `term_element['name']` to cover the cases with missing namespaces if they will be present based on this [PR](https://github.com/openstax/kitchen/pull/465/files)):

Reply:
Is it necessary? Can't we use just `term_element.key?('name')` ? :thinking: I think we need cases with different namespaces only when we want to take a value of the key (like in line 172). But if not, if it has to be specified also in condition, should we change line 167 as well?

Reply:
Good point Natalia! I have missed it. The namespaces don't have to be specified in the condition.